bridges/snowbridge/pallets/system/src/lib.rs

```diff
@@ -574,6 +594,57 @@ pub mod pallet {
 			});
 			Ok(())
 		}
+
+		/// Sends a message to the Gateway contract to register a new
+		/// token that represents `asset`.
+		///
+		/// - `origin`: Must be `MultiLocation` from sibling parachain
+		#[pallet::call_index(10)]
+		#[pallet::weight(T::WeightInfo::register_token())]
+		pub fn register_token(
+			origin: OriginFor<T>,
+			asset: Box<VersionedLocation>,
+			metadata: AssetRegistrarMetadata,
+		) -> DispatchResult {
+			let origin_location: Location = T::SiblingOrigin::ensure_origin(origin)?;
+
+			let (para_id, agent_id) = ensure_sibling::<T>(&origin_location)?;
+
+			let asset: Location =
+				(*asset).try_into().map_err(|_| Error::<T>::UnsupportedLocationVersion)?;
+
+			Self::do_register_token(para_id, agent_id, asset, metadata)?;
+
+			Ok(())
+		}
+
+		/// Sends a message to the Gateway contract to register a new
+		/// token that represents `asset`.
+		///
+		/// - `origin`: Must be `MultiLocation`
+		#[pallet::call_index(11)]
+		#[pallet::weight(T::WeightInfo::force_register_token())]
+		pub fn force_register_token(
```

Comment:
I don't think we need a `force` variant for registering tokens.

All the other `force_*` governance methods are so that polkadot governance can administer all the system parachains.  

However I don't see system parachains needing to register tokens.

Reply:
> However I don't see system parachains needing to register tokens.

I would assume relay token(e.g. DOT) is registered from AH, more context in https://github.com/Snowfork/polkadot-sdk/pull/128#discussion_r1558784632